<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyMaskingPIFormClass</class>
 <widget class="GUIFormBase" name="PropertyMaskingPIFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertyMaskingPIForm</string>
  </property>
  <widget class="QToolButton" name="toolButtonIneffective">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>無効</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetMaskingList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>571</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Effective</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Limited</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Area</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="toolButtonEffective">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>有効</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonGenerateMaskFromSameColor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>650</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>同一輝度からのマスク生成</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyMaskingPI.qrc">
     <normaloff>:/Qtres/Add.png</normaloff>:/Qtres/Add.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyMaskingPI.qrc"/>
 </resources>
 <connections/>
</ui>
